go/integration_test/interpreter/vm_test.go

```diff
@@ -41,7 +41,10 @@ var (
 func TestExamples_ComputesCorrectResult(t *testing.T) {
 	for _, example := range testExamples {
 		for _, variant := range getAllInterpreterVariantsForTests() {
-			vm := tosca.GetInterpreter(variant)
+			vm, err := tosca.NewInterpreter(variant)
+			if err != nil {
+				t.Fatalf("failed to load %s interpreter: %v", variant, err)
```

Comment:
nit: this  error message is used in multiple files, it could be oursourced to `makeLoadError(variant, err)`

Reply:
This is an error message of a unit test, not an error. I think it is fine to be explicit there and not to obfuscate error messages through level of indirection.